site/src/modules/notifications/NotificationsInbox/InboxAvatar.tsx

```diff
@@ -0,0 +1,30 @@
+import { Avatar } from "components/Avatar/Avatar";
+import {
+	InfoIcon,
+	LaptopIcon,
+	LayoutTemplateIcon,
+	UserIcon,
+} from "lucide-react";
+import type { FC } from "react";
+import type React from "react";
+
+export type InboxIcon =
+	| "DEFAULT_WORKSPACE_ICON"
+	| "DEFAULT_ACCOUNT_ICON"
+	| "DEFAULT_TEMPLATE_ICON"
+	| "DEFAULT_OTHER_ICON";
+
+const inboxIcons: Record<InboxIcon, React.ReactNode> = {
+	DEFAULT_WORKSPACE_ICON: <LaptopIcon />,
+	DEFAULT_ACCOUNT_ICON: <UserIcon />,
+	DEFAULT_TEMPLATE_ICON: <LayoutTemplateIcon />,
+	DEFAULT_OTHER_ICON: <InfoIcon />,
+};
+
+type InboxAvatarProps = {
+	icon: InboxIcon;
+};
+
+export const InboxAvatar: FC<InboxAvatarProps> = ({ icon }) => {
+	return <Avatar variant="icon">{inboxIcons[icon]}</Avatar>;
```

Comment:
We need to make it future proof, so theoretically icon can be either a custom image link or "DEFAULT_ICON_*"

Reply:
Since we don't have plans for that I would not add that. I really like to avoid adding code to handle scenarios were are not planned to happen any time soon.

Reply:
Actually this is how backend is implemented, so when the notification icon has custom url, backend will return it. Frontend must be compliant here.